lib/oppen/print_stack.rb

```diff
@@ -245,6 +250,19 @@ def write(obj)
       buffer.write(obj.to_s)
     end
 
+    # Erase the last `count` characters.
+    #
+    # @param count [Integer]
+    #
+    # @return [Nil]
+    def erase(count = 0)
+      return if !count
```

Comment:
Really?  This is checking for falsies: `nil` and `false`.  I think both should be invalid and make errors.  Did you mean to return if `count == 0`?

Reply:
There was a time when we passed nil to the method, but we decided to remove this feature. I will make the change.